.github/workflows/test-and-deploy.yml

```diff
@@ -27,6 +27,30 @@ jobs:
       - name: Build & Test
         run: make test
 
+  sonarCloud:
```

Comment:
nit: Don't think this should be run on tag pushes and as part of the cron since it already runs for all branches. Just adds extra time. Could move into a separate workflow which just runs on PRs and branches.

Reply:
Will update